jikkou/kafka-topics-values.yaml

```diff
@@ -82,6 +82,12 @@ apps:
       - topic.OdeRawEncodedPSMJson
       - topic.OdePsmTxPojo
       - topic.OdePsmJson
+      - topic.OdeTimRxJson
+      - topic.OdeDNMsgJson
+      - topic.OdeBsmDuringEventPojo
+      - topic.OdeBsmRxPojo
+      - topic.OdeBsmTxPojo
+      - topic.OdeTimBroadcastPojo
```

Comment:
ode-svcs application.yaml shows several of these as "disabled", are they needed?

    disabled-topics:
      - "topic.OdeBsmRxPojo"
      - "topic.OdeBsmTxPojo"
      - "topic.OdeBsmDuringEventPojo"
      - "topic.OdeTimBroadcastPojo"

Reply:
I noticed that "topic.OdeTimRxJson" was being used so I was thinking of adding them all, but I could remove the disabled topics.

Reply:
No string opinion, I just noticed that.